website_event_snippet_calendar/static/src/js/animation.js

```diff
@@ -147,22 +132,25 @@ odoo.define('website_event_snippet_calendar.animation', function (require) {
         },
 
         render_calendar: function () {
-            this.$calendar.empty().datetimepicker(this.datepicker_options);
+            var enabledDated = _.map(this._dates.matches, function (ndate) {
```

Comment:
Do you have a typo here? Or was it in purpose? :thinking: 

```suggestion
            var enabledDates = _.map(this._dates.matches, function (ndate) {
```